config/sync/migrate_plus.migration.stanford_events_importer.yml

```diff
@@ -0,0 +1,163 @@
+uuid: f12377a6-f9c4-4050-8493-87df53c70bae
+langcode: en
+status: true
+dependencies: {  }
+_core:
+  default_config_hash: cGwmz8ZjzFcJd_ygOCdIMWZpXlLTbA3JM3CN-rniTws
+id: stanford_events_importer
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags: null
+migration_group: stanford_events
+label: 'Events importer'
+source:
+  track_changes: true
+  plugin: url
+  data_fetcher_plugin: file
+  urls: null
+  data_parser_plugin: simple_xml
+  item_selector: /EventList/Event
+  fields:
+    -
+      name: guid
+      label: GUID
+      selector: guid
+    -
+      name: title
+      label: 'Event Title'
+      selector: title
+    -
+      name: body
+      label: Description
+      selector: description
+    -
+      name: event_start
+      label: 'Event start'
+      selector: isoEventDate
+    -
+      name: event_end
+      label: 'Event end'
+      selector: isoEventEndDate
+    -
+      name: link
+      label: 'Event Link'
+      selector: link
+    -
+      name: audience
+      label: 'Event Audience'
+      selector: audiences
+    -
+      name: categories
+      label: Categories
+      selector: categories
+    -
+      name: location
+      label: Location
+      selector: locationText
+    -
+      name: map_link
+      label: 'Map Link'
+      selector: mapUrl
+    -
+      name: event_sponsor
+      label: Sponsor
+      selector: sponsor
+    -
+      name: contact_email
+      label: 'Contact Email'
+      selector: contactEmail
+    -
+      name: contact_phone
+      label: 'Contact Phone'
+      selector: contactPhone
+  ids:
+    guid:
+      type: string
+  constants:
+    status: 1
+    type: stanford_event
+    stanford_html: stanford_html
+    stanford_minimal_html: stanford_minimal_html
+    map_title: 'Open in map'
+process:
+  status: constants/status
+  type: constants/type
+  title: title
+  body/value: body
+  body/format: constants/stanford_html
+  su_event_email: contact_email
+  su_event_telephone: contact_phone
+  su_event_sponsor:
```

Comment:
ever multiple sponsors from the feed?

Reply:
It does appear so. Thanks, I'll make that change.